.circleci/config.yml

```diff
@@ -808,7 +808,8 @@ jobs:
       - run-tests-linux:
           # some native-dependent tests fail because of the lack of native
           # headers on emsdk-bundled clang
-          test_targets: "other skip:other.test_native_link_error_message"
+          test_targets: "
+            other.test_minimal_runtime_code_size_hello_embind"
```

Comment:
I guess you can revert this file?

Reply:
Oh I thought I reverted all CI testing changes but this remained... Reverted.